src/eye_extractor/dr/binary_vars.py

```diff
@@ -29,12 +29,6 @@
         r')\b',
         re.I
     )),
-    ('venbeading', re.compile(

```

Comment:
Removing these broke a few tests. I removed or migrated them (if they didn't exist yet in the target environment).